### ISSUE TITLE
evaluation metric

### ISSUE THREAD

**Issue by k5yisen** [rodrigo2019/keras-yolo2#15]
Hi Rodrigo, 

I have yolo boxes format for annotations. I don't have XML annotations and CSV annotations.  How can I calculate my evaluation metric?

**Comment by rodrigo2019** [comment]
You need the ground truth annotations to compare with your yolo output

**Comment by k5yisen** [comment]
I have ground-truth annotations type(txt). I originally trained my data with [https://github.com/experiencor/keras-yolo2] I need evaluation metrics to compare my results. I am currently implementing your evaluate.py code. How can I get my yolo output?

**Comment by rodrigo2019** [comment]
https://github.com/rodrigo2019/keras_yolo2/blob/master/keras_yolov2/frontend.py#L215
